assets/scss/_code.scss

```diff
@@ -61,3 +61,7 @@
         padding: $pre-padding;
     }
 }
+
+.medium-zoom-overlay, .medium-zoom-image--opened {
+    z-index: 2000 !important;
+}
```

Comment:
NIT: remove new line.

Reply:
There seems to be an extra line after 67, but it's non blocking. I will approve. Tthis is good stuff ...

Reply:
Thank you so much for the PR merge